src/log/__init__.py

```diff
@@ -11,7 +11,7 @@ def setup_logger():
     timefmt = '%y%m%d_%H:%M:%S'
 
     logging.basicConfig(
-        filename=f'{root_dir}/{time.strftime("algo-trader.%y%m%d_%H%M%S.log")}',
+        filename=f'{root_dir}/../logs/{time.strftime("algo-trader.%y%m%d_%H%M%S.log")}',
```

Comment:
if there is no `logs` directory, the app will fail to load as it can't create the log file. 
Adding a check for the existence of the directory and creating it if needed will resolve that.

Reply:
Good point, I've added a `.gitkeep` file to the `logs` folder so that git should create one, however if you accidently delete the folder, this new commit should recreate it per your suggestion. I've also gone ahead and renamed `root_dir` to `log_dir` since this is not used in multiple places and we can do without the path traversal in the config.